### PR TITLE
Added search rep to registry

### PR DESCRIPTION
**Pull request by cohnjesse**
## Description
[Original Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/17743)

Now that the search for representative app is getting further along we want to add it to staging so that stakeholders can see it.

## Acceptance criteria
- [x] Added the search for represetnative app to content-build repo

### CODE REVIEW

File: src/applications/registry.json [hunk 1]
```diff
@@ -1139,5 +1139,14 @@
       "vagovprod": false,
       "layout": "page-react.html"
     }
+  },
+  {
+    "appName": "Search Representative",
```
Comment (alexandec) [inline src/applications/registry.json:1144]: There's no `entryName` property here. Without that, the content build won't be able to find the assets for your app. You can check out the registry entries for the other apps to see examples. See also the [docs for creating a new app](https://depo-platform-documentation.scrollhelp.site/developer-docs/Creating-a-new-application.1847525496.html)

Comment (cohnjesse) [reply src/applications/registry.json:1144]: No worries, added this in

Comment (timwright12) [reply src/applications/registry.json:1144]: make sure that vets-website assets for this app are deployed before merging or this won't pass the build

Comment (theodur) [reply src/applications/registry.json:1144]: Also, you don't need to keep the `entryFile` property here

**Review by alexandec — APPROVED** [review]

File: src/applications/registry.json (hunk 1 above)
Comment (cohnjesse) [reply src/applications/registry.json:1144]: @timwright12 Yep this app has been in `vets-website` for a while just not on staging.

**Pull request merged**